lib/utils/lnurl.dart

```diff
@@ -20,11 +20,19 @@ bool isLNURL(String url) {
   return false;
 }
 
-bool isLightningAddress(String email) {
-  // This is just a normal e-mail address.
+// FIXME(nochiel) Maybe this should parse 'lightning:' uris and return the address instead of a boolean?
```

Comment:
I think it is a good idea to change the method to return the parsed address and null if there isn't a match.
It will save a lot of code above that always checks `startsWith('lightning:')`